task/make.js

```diff
@@ -0,0 +1,14 @@
+function make(...params) {
+  const values = [];
+  function foo(...args) {
+    if (args[0] instanceof Function) {
+      const func = args[0];
+      return values.reduce((accum, elem) => func(accum, elem));
```

Comment:
Looks like it wouldn't be working in calling without number arguments (in case make(sum)).